include/mtx/events/event_type.hpp

```diff
@@ -131,6 +131,8 @@ enum class EventType
     FullyRead,
     //! m.direct
     Direct,
+    //! m.ignored_user_list
+    IgnoredUsers,
```

Comment:
In general we try to name those after the event type, so this would be IgnoredUserList, but otoh it doesn't really matter here :)

Reply:
Ahh right, sorry about that.